src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/ApprovedPremisesBookingSeedJob.kt

```diff
@@ -80,6 +80,8 @@ class ApprovedPremisesBookingSeedJob(
   )
 
   override fun processRow(row: ApprovedPremisesBookingSeedCsvRow) {
+    Thread.sleep(1_000)
```

Comment:
Just wondering, is there an endpoint to bulk-fetch CRNs from Delius via CommunityAPI?

Reply:
Doesn't look like it unfortunately: https://github.com/ministryofjustice/community-api/blob/main/src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

Reply:
Oh boo - I might ask the question. Would help massively with our preemptive caching too